src/01-highlights-list/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * Highlights List
+ * A stylised list displaying bullet points and a title
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType, registerBlockStyle } from '@wordpress/blocks';
+import { RichText } from '@wordpress/block-editor';
+
+registerBlockType('mojblocks/highlights-list', {
+    title: __('Highlights List', 'mojblocks'),
+    icon: 'list-view',
+    category: 'mojblocks',
+    example: {
+        attributes: {
+            listTitle: 'This is a highlights list title',
+            listText: 'This is a list item',
+        },
+    },
+    attributes: {
+        listTitle: {
+            type: 'string',
+            source: 'html',
+            selector: '.mojuk-highlights-list__heading-text'
+        },
+        listText: {
+            type: 'array',
+            source: 'children',
+            multiline: 'li',
+            selector: '.mojuk-highlights-list__content > ul'
+        }
+    },
+
+    edit: props => {
+        let {
+            attributes: {
+                listTitle,
+                listText
+            },
+            className,
+            setAttributes
+        } = props;
+
+        // Grab newListTitle, set the value of listTitle to newListTitle.
+        let onChangeListTitle = newListTitle => {
+            setAttributes({ listTitle: newListTitle });
+        };
+
+        // Grab newListText, set the value of listText to newListText.
+        let onChangeListText = newListText => {
+            setAttributes({ listText: newListText });
+        };
+
+        return (
+            <div className={`${className}`}>
+                <div className="mojuk-highlights-list__heading-container">
+                    <h3 className="mojuk-highlights-list__heading">
+				  <span role="text">
+					<span className="mojuk-highlights-list__heading-text">
+					  <RichText
+                          placeholder={__('Highlights title', 'mojblocks')}
+                          value={listTitle}
+                          onChange={onChangeListTitle}
+                      />
+					</span>
+				  </span>
+                    </h3>
+                </div>
+                <div className={'mojuk-highlights-list__content'}>
+                    <RichText
+                        tagName='ul'
+                        multiline='li'
+                        placeholder={__('Highlights item', 'mojblocks')}
+                        onChange={onChangeListText}
+                        value={listText}
+                    />
+                </div>
+            </div>
+        );
+    },
+
+    save: props => {
+        let {
+            attributes: {
+                listTitle,
+                listText
+            }
+        } = props;
+
+        return (
+            <div className="mojuk-grid-column-width mojuk-highlights-list--type">
+                <div className="mojuk-highlights-list__heading-container">
+                    <h3 className="mojuk-highlights-list__heading">
+				 <span role="text">
```

Comment:
Ooh - what's the rationale behind `role="text"`? I've not seen that one before so I had a bit of a read up, and it looks like it's not in the [official ARIA spec at present](https://www.w3.org/WAI/PF/aria/roles#textbox). It looks like the [Working Group have considered adding it in once or twice](https://github.com/w3c/aria/issues/870), but it seems to be still a bit of a debate. It's an interesting one!

I'd say we maybe don't need it here as I'm assuming the RichText component wrappers everything in the right semantic HTML, and [the first rule of ARIA is don't use it](https://www.w3.org/TR/using-aria/#firstrule) - but that's an assumption on my part! What are your thoughts?

Reply:
This is up for debate, I took the role from the NHS plugin. My feeling is we build the components and consider accessibility during build but then discuss accessibly in a dev meet once all 7 have been created to address any concerns. Does that sound good?